index.html

```diff
@@ -1430,7 +1430,7 @@ <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="cont
      <dt>This version:
      <dd><a class="u-url" href="https://wicg.github.io/animation-worklet/">https://wicg.github.io/animation-worklet/</a>
      <dt>Issue Tracking:
-     <dd><a href="https://github.com/WICG/animation-worklet/issues/">GitHub</a>
+     <dd><a href="https://github.com/majido/animation-worklet/issues/">GitHub</a>
```

Comment:
Looks like your bikeshed build is accidentally changing the source repo location? This breaks links to issues.